aws-iot-jobtemplate/src/main/java/software/amazon/iot/jobtemplate/Translator.java

```diff
@@ -0,0 +1,147 @@
+package software.amazon.iot.jobtemplate;
+
+import software.amazon.awssdk.services.iot.model.AbortConfig;
+import software.amazon.awssdk.services.iot.model.JobExecutionsRolloutConfig;
+import software.amazon.awssdk.services.iot.model.PresignedUrlConfig;
+import software.amazon.awssdk.services.iot.model.RateIncreaseCriteria;
+import software.amazon.awssdk.services.iot.model.TimeoutConfig;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Translator {
+
+    static software.amazon.awssdk.services.iot.model.AbortConfig getAbortConfig(ResourceModel model) {
+        if(model == null || model.getAbortConfig() == null || model.getAbortConfig().getCriteriaList().isEmpty()) {
+            return null;
+        }
+        List<software.amazon.iot.jobtemplate.AbortCriteria> listModelCriteria = model.getAbortConfig().getCriteriaList();
+
+        List<software.amazon.awssdk.services.iot.model.AbortCriteria> listSdkCriteria = listModelCriteria.stream().map( modelCriteria -> {
```

Comment:
Can we use a unified name for the term? I prefer to put List at the end of string like the one on line 39.